src/xGadgetron/pGadgetron/Gadgetron.py

```diff
@@ -724,6 +843,13 @@ def __calc_from_images(self, data, method_name):
 DataContainer.register(CoilSensitivityData)
 
 
+class ContiguousError(ValueError):
+    """
+    ValueError for discontiguous memory as per
+    https://data-apis.org/array-api/latest/API_specification/generated/array_api.asarray.html
+    """
+
+
```

Comment:
remove and import https://github.com/SyneRBI/SIRF/blob/8fbfe5294e4efd57d12ae15d87b97f64e96d09d4/src/common/SIRF.py#L47

Reply:
done